metadata/adaptors.go

```diff
@@ -47,11 +48,11 @@ func adaptImage(o interface{}) filters.Adaptor {
 				return obj.Target.Digest.String(), len(obj.Target.Digest) > 0
 			case "mediatype":
 				return obj.Target.MediaType, len(obj.Target.MediaType) > 0
+			case "size":
+				return strconv.FormatInt(obj.Target.Size, 10), true
```

Comment:
This is slightly disconnected from what is shown on the output. This ends up just being the manifest size and not the `SIZE` field.

Reply:
Having `target.size` may still be useful, but I don't think we can qualify that is satisfying the below TODO statement. That would require calling `image.Size`.

Reply:
Ah, I didn't notice that. I think I can have `case "size":` as like `case "name":` above.

Reply:
Hmm, this actually could be a show stopper. image.Size() needs Content Store, but images.Store shouldn't know about Content Store I think.

Reply:
I agree its non-trivial, my point is more just to leave the comment in regards to this PR. I'm not sure `target.size` ends up being that useful though. Getting this PR in just for the content size filtering is fine by me as well.